### PR TITLE
[EDU-5467] Translate Edge SQL guides

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Translates Edge SQL guides.

### CODE REVIEW

File: src/content/docs/pt-br/pages/store-jornada/sql/criar-banco.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+permalink: /documentacao/produtos/store/sql/criar-banco-de-dados/
+meta_tags: >-
+  Edge sql, sql, database, CRUD, API
+namespace: docs_store_journey_sql_create_database
+menu_namespace: storeMenu
+---
+
+Este guia demonstra como usar o **Edge SQL** da Azion para criar um novo banco de dados usando a API da Azion.
+
+## Criando um banco de dados
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/criar-banco.mdx:14]:
```suggestion
## Crie um banco de dados
```

File: src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+permalink: /documentation/products/store/sql/create-table/
+meta_tags: >-
+  Edge SQL, SQL database, create table, database management, CRUD operations, API integration, data storage, SQL tutorial, Edge SQL guide, database creation, SQL commands
+namespace: docs_store_journey_sql_create_table
+menu_namespace: storeMenu
+---
+
+import LinkButton from 'azion-webkit/linkbutton'
+
+Após ter criado seu [banco de dados Edge SQL](/pt-br/documentacao/produtos/store/sql/criar-banco-de-dados//), você pode começar a criar tabelas e inserir dados nelas.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx:14]:
```suggestion
Após ter criado seu [banco de dados do Edge SQL](/pt-br/documentacao/produtos/store/sql/criar-banco-de-dados//), você pode começar a criar tabelas e inserir dados nelas.
```

File: src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+<LinkButton link="/pt-br/documentacao/produtos/store/edge-sql/" label="Consultar referência do Edge SQL" severity="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx:16]:
```suggestion
<LinkButton link="/pt-br/documentacao/produtos/store/edge-sql/" label="Saiba mais sobre o Edge SQL" severity="secondary" />
```

File: src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Criando uma tabela
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/criar-tabela.mdx:18]:
```suggestion
## Crie uma tabela
```

File: src/content/docs/pt-br/pages/store-jornada/sql/deletar-banco.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+permalink: /documentacao/produtos/store/sql/deletar-database/
+meta_tags: >-
+  Edge SQL, SQL, database, delete, API
+namespace: docs_store_journey_sql_delete_database
+menu_namespace: storeMenu
+---
+
+Este guia demonstra como usar o **Edge SQL** da Azion para excluir um banco de dados existente usando a API da Azion.
+
+## Excluindo um banco de dados
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/deletar-banco.mdx:14]:
```suggestion
## Exclua um banco de dados
```

File: src/content/docs/pt-br/pages/store-jornada/sql/deletar-registro.mdx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+title: Como excluir um registro em um banco de dados do Edge SQL
+description: >-
+  Veja instruções passo a passo para excluir um registro em um banco de dados Edge SQL para gerenciamento eficiente de dados.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/deletar-registro.mdx:4]:
```suggestion
  Veja instruções passo a passo para excluir um registro em um banco de dados do Edge SQL para gerenciamento eficiente de dados.
```

File: src/content/docs/pt-br/pages/store-jornada/sql/deletar-registro.mdx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+permalink: /documentacao/produtos/store/sql/deletar-registro/
+meta_tags: >-
+  Edge SQL, SQL database, delete record, database management, CRUD operations, API integration, data deletion, SQL tutorial, Edge SQL guide, database modification, SQL commands
+namespace: docs_store_journey_sql_delete_record
+menu_namespace: storeMenu
+---
+
+Esta guia mostra como usar o **Edge SQL** da Azion para excluir um registro em seu banco de dados usando a API da Azion.
+
+## Excluindo um registro
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/deletar-registro.mdx:14]:
```suggestion
## Exclua um registro
```

File: src/content/docs/pt-br/pages/store-jornada/sql/inserir-em-tabela.mdx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+permalink: /documentacao/produtos/store/sql/inserir-em-tabela/
+meta_tags: >-
+  Edge SQL, SQL, database, CRUD, API, insert data
+namespace: docs_store_journey_sql_insert_into_table
+menu_namespace: storeMenu
+---
+
+Se você já criou um banco de dados, você pode usar o **Edge SQL** da Azion para inserir novos dados através da API da Azion.
+
+## Adicionando dados a uma tabela
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/inserir-em-tabela.mdx:14]:
```suggestion
## Adicione dados a uma tabela
```

File: src/content/docs/pt-br/pages/store-jornada/sql/listar-db.mdx [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+permalink: /documentacao/produtos/store/sql/listar-bancos-de-dados/
+meta_tags: >-
+  Edge sql, sql, database, CRUD, API
+namespace: docs_store_journey_sql_list_databases
+menu_namespace: storeMenu
+---
+
+Este guia demonstra como usar o **Edge SQL** da Azion para listar os bancos de dados relacionados à sua conta usando a API da Azion.
+
+## Listando bancos de dados
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/listar-db.mdx:14]:
```suggestion
## Liste bancos de dados
```

File: src/content/docs/pt-br/pages/store-jornada/sql/query-rows.mdx [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+menu_namespace: storeMenu
+---
+
+import LinkButton from 'azion-webkit/linkbutton'
+
+Depois de criar seu banco de dados do Edge SQL, você pode recuperar os dados armazenados nele.
+
+<LinkButton link="/pt-br/documentacao/produtos/store/edge-sql/" label="Consulte a referência do Edge SQL" severity="secondary" />
+
+## Listando dados de uma tabela
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/store-jornada/sql/query-rows.mdx:16]:
```suggestion
## Liste dados de uma tabela
```

**Review by MarianaAguilera — APPROVED** [review]
minor style suggestions

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**